### DISCUSSION TITLE
How to upgrade to Latest Laravel-BoilerPlate

### DISCUSSION THREAD

**Discussion by faisaljawad1** [rappasoft/laravel-boilerplate#1573] [Q&A]
I'm currently using BoilerPlate with Laravel 6.x but I plan to switch from this one to the latest either 8.x or 9.x whatever is available at that time.

Can you please tell me what will be the process of upgrading the boiler-plate?

**Comment by bryanrscott** [comment]
For me it has always been a manual update, starting with a fresh project.   But many things are quite easy to manually migrate do to the structure and can be just copied across.  eg \app\Domain\ ,  \routes\frontend\.  etc....

**Reply by faisaljawad1** [reply]
@bryanrscott Thanks Bryan but my requirement was to **upgrade the Laravel version** mainly and I was able to achieve it by upgrading the **composer.json** step by step.
I simply updated the composer packages as per Laravel's official documentation for each version.

**Sources:**
1. https://laravel.com/docs/7.x/upgrade
2. https://laravel.com/docs/8.x/upgrade

I hope it turns out helpful for other people.

**Comment by bryanrscott** [comment]
If your looking to do just a laravel version update laravel shift is paid service (i think about $15USD) which does a pretty good job at automating

**Reply by faisaljawad1** [reply]
I saved $15USD 😃 
Did it all myself through composer.json and was able to achieve the latest stable Laravel version within the same application.
